docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -1,10 +1,22 @@
 [[recovery]]
 === Indices Recovery
 
-The following _expert_ setting can be set to manage the recovery policy.
+_Recovery_ is the process used to build a new copy of a shard on a node by
+copying data from the primary. {es} uses this recovery process to rebuild shard
+copies that were lost if a node has failed, and uses the same process when
+migrating a shard copy between nodes to rebalance the cluster or to honour any
```

Comment:
NIT: I'm sorry :) but maybe `honor` (we're consistently using :us: in our docs as far as I can see)?